editoast/Dockerfile

```diff
@@ -23,11 +23,15 @@ COPY --from=static_assets . /assets
 #######################
 # Build assets        #
 #######################
-FROM alpine:latest AS editoast_assets
+FROM ubuntu:latest AS editoast_assets
```

Comment:
Why the switch of image from `alpine` to `ubuntu`?

Reply:
Because the binary has some library bindings that don’t work out of the box on alpine. I don’t know if it’s a musl/gnu thing or some library that is install on the ubuntu image

Reply:
We could also use `FROM chef` and then simply install it with `cargo`.

Reply:
I’m fine with either, do you have any preference?